Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,17 @@
+.title {  
+  font-size: $text-big;
+  position: relative;
+  margin: 40px 0 0 50px;
```

Comment:
use padding instead here, this is because this margin is to make the image be inside the element bounds, those bounds end when the margin starts. so it's better to use padding in that case.